@AH-64D Apache Longbow/Addons/fza_ah64_fcr/functions/fn_update.sqf

```diff
@@ -67,28 +67,20 @@ private _fcrTargets  = [];
     if ([_target] call fza_fnc_targetIsADA) then { _type = FCR_TYPE_ADU; };
 
     if ((_type != FCR_TYPE_FLYER && _type != FCR_TYPE_HELICOPTER) && _fcrMode == 2) then {continue;};
-    if ((vectorMagnitude velocityModelSpace _target) < 5 && _fcrMode == 2) then {continue;};
+    //if ((vectorMagnitude velocityModelSpace _target) < 5 && _fcrMode == 2) then {continue;};
```

Comment:
If line not needed, then just remove

Reply:
was commented out for testing - corrected